frontend/src/app/features/work-packages/components/wp-custom-actions/wp-custom-actions/wp-custom-action.component.ts

```diff
@@ -80,7 +85,7 @@ export class WpCustomActionComponent {
       lockVersion: this.workPackage.lockVersion,
       _links: {
         workPackage: {
-          href: this.workPackage.href,
+          href: this.v3.workPackagePath(this.workPackage.id!),
```

Comment:
I'm totally fine with adding this path, but theoretically it's duplicated from `this.apiV3Service.work_packages(this.workPackage.id).path`

Reply:
Thanks Oliver, I discovered that we have `this.workPackage.previewPath()` which seems to be shorter. I replaced with that.